tests/zap.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import infra.network
+import infra.proc
+import infra.net
+import suite.test_requirements as reqs
+import infra.e2e_args
+import subprocess
+
+
+@reqs.description("HTTP fuzzing with ZAP")
+@reqs.at_least_n_nodes(1)
+def test(network, args):
+    node = network.nodes[0]
+    openapi_endpoint = f"https://{node.host}:{node.rpc_port}/node/api"
+
+    args = [
+        "docker", "run", "--rm", "--network", "host", 
+        "-v", f"{args.binary_dir}:/zap/wrk",
+        "-t", "owasp/zap2docker-stable", "zap-api-scan.py",
+        "-t", openapi_endpoint, "-f", "openapi",
+        "-c", "zap.config",
+        "-l", "INFO", "-r", "zap_report.html", "-J", "zap_report.json"
+    ]
+
+    r = subprocess.run(args, check=False)
```

Comment:
check=True, then we don't need the assert?

Reply:
Sure, I just copied the tlstest, no idea why it was like that.